### PR TITLE
Feat: update tvl fetched from staking pools / vault / vesting / lock …

### PR DESCRIPTION
**Pull request by DramaCrypto**
…contracts

**NOTE**

> If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).

1. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
2. The protocol is usually listed within 24 hours of merging the PR
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/candycity exports TVL: 
        
 
 
    cronos                    3.14 M
    
    total                    3.14 M

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
hi @DramaCrypto 

Sorry, but we cant count candy tokens under tvl.
We can count the user staked tokens under 'staking'
and the vested candy tokens under 'vesting'

https://docs.llama.fi/list-your-project/what-to-include-as-tvl

**Comment by llamatester** [comment]
The adapter at projects/candycity exports TVL: 
        
 
 
    cronos-vesting            1.06 M
    cronos-staking            2.31 M
    cronos                    3.47 M
    vesting                   1.06 M
    staking                   2.31 M
    
    total                    3.47 M

**Comment by DramaCrypto** [comment]
I added vesting and staking as you commented @g1nt0ki

File: projects/candycity/index.js [hunk 1]
```diff
@@ -1,10 +1,60 @@
...
+};
+
 module.exports = {
   misrepresentedTokens: true,
   methodology:
-    "Factory address (0x84343b84EEd78228CCFB65EAdEe7659F246023bf) is used to find the LP pairs. TVL is equal to the liquidity on the AMM.",
+    "Factory address (0x84343b84EEd78228CCFB65EAdEe7659F246023bf) is used to find the LP pairs. TVL is equal to the liquidity on the AMM and the candy tokens in the staking pools / vault / vesting contract / lock contract.",
   cronos: {
-    tvl: getUniTVL({ factory: '0x84343b84EEd78228CCFB65EAdEe7659F246023bf', chain: 'cronos', useDefaultCoreAssets: true }),
+    tvl:  sdk.util.sumChainTvls([chainTvl, fetchTvl]),
```
Comment (g1nt0ki) [inline projects/candycity/index.js:55]: @DramaCrypto changed this line to `tvl:  chainTvl,` then it is good, else staking would be double counted

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: projects/candycity/index.js (hunk 1 above)
Comment (g1nt0ki) [reply projects/candycity/index.js:55]: I will fix this

**Pull request merged**